login_style.css

```
@@ -0,0 +1,139 @@
+:root {
+  font-size: 16px;
+}
+
+body {
+  width: 100vw;
+}
+/* 공통 */
+.container {
+  display: flex; /* 보기 방식은 flex로*/
+  flex-direction: column; /*흐름의 방향은 위에서 아래로*/
+  justify-content: center; /* 기본축 방향에서 중간에 위치하게*/
+  align-items: center; /* 교차축 방향에서도 중간에 위치하게*/
+}
+
+  /* 상단바  */
+.gnb{
+  max-width : 640px;
+  height : 70px;
+  margin : 50px auto;
+  padding : 0 122px;
+  background-color : #ffffff;
+  position:relative;
+  top: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.gnb-inner_Frame{
+  width : 396px;
+  height : 132px;
+}
+/* 중간 내용물 */
+.main{
+  width : 640px;
+  height: 430px;
+  margin : 50px 0 0 0;
```

top을 제외한 다른 값들이 모두 선언될 필요가 없는거라면 이게 더 나을것같네요 
```suggestion
  margin-top : 50px;
```